pkg/server/application_api/storage_inspection_test.go

```diff
@@ -217,6 +217,11 @@ func TestAdminAPIDataDistribution(t *testing.T) {
 		post_id INT REFERENCES roachblog.posts,
 		body text
 	)`)
+
+	// Test for null raw sql config column in crdb_internal.zones,
+	// see: https://github.com/cockroachdb/cockroach/issues/140044
+	//sqlDB.Exec(t, `-- ALTER TABLE roachblog.posts CONFIGURE ZONE = ''`)
```

Comment:
There's a bunch of commented out code in this test. Did you intend to merge this? I think it's fine to merge for now and then do a separate update to remove/uncomment if you like.

Reply:
Oops this was my bad - meant to put this back, I had commented it out locally to test something and didn't mean to push. I'll follow up with a PR.

Reply:
https://github.com/cockroachdb/cockroach/pull/140621